src/ui/IngredientForm.tsx

```diff
@@ -0,0 +1,36 @@
+import { FieldArray } from 'formik';
+import Ingredient from './Ingredient';
+import styled from '@emotion/styled';
+import { FC } from 'react';
+
+const StyledFieldArray = styled(FieldArray)`
+  background: var(--background);
+  border: 1px var(--text) solid;
+  color: var(--text);
+  height: 51px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const IngredientForm: FC = ({ props }) => {
```

Comment:
hehe this had me confused for sec. never name a prop "props". here the prop name should be values.

the first param of an FC is props by default. You should be typing your props (see other comment). Then destructuring the props here

Reply:
some props are added by default and they don't have to be typed. e.g. ({ children })